pytradfri/device/air_purifier.py

```diff
@@ -92,9 +92,13 @@ def filter_runtime(self) -> int:
         return self.raw.filter_runtime
 
     @property
-    def filter_status(self) -> int:
-        """Return filter status."""
-        return self.raw.filter_status
+    def filter_status(self) -> bool:
+        """
+        Return True if filter needs to be replaced.
+
+        This property is true when filter_lifetime_remaining is less than zero.
+        """
+        return bool(self.raw.filter_status)
```

Comment:
This is a breaking change although minor I would say.

Reply:
So, a major version bump in a follow up?

Reply:
Yeah, good to stay consistent with semver.